packages/libs/deer/src/impls/core/mod.rs

```diff
@@ -0,0 +1,21 @@
+mod array;
+mod atomic;
+mod bool;
+mod floating;
+mod integral;
+mod non_zero;
+mod string;
+mod unit;
+
+// DONE: f32, f64, i*, u*, char, str, array, Atomic*, NonZero, Unit
+// core:
+// tuple, LazyCell, OnceCell, SyncUnsafeCell, Cel, Ref,
+// RefCell, RefMut, UnsafeCell, PhantomData, ManuallyDrop, Option, Result, Wrapping, Bound, Range,
+// Ip*, Reverse, PhantomData
+
+// alloc:
+// Cow, Box, ThinBox, BinaryHeap, BTreeMap, BTreeSet, LinkedList, VecDequeue, Vec, CString, Rc,
+// String, Arc
+
+// std:
+// HashMap, Path, Barrier(?), Mutex, Once, RwLock
```

Comment:
Do you want to keep this around or is it superfluous?

Reply:
This is indeed redundant, as this is the (outdated) predecessor of #1700, removed in [0a2954a](https://github.com/hashintel/hash/pull/1516/commits/0a2954a2bb6f027e25149e0825d6dccddbe9a2a3)